hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -221,8 +234,25 @@ public boolean commitStats(String instantTime, List<HoodieWriteStat> stats,
                              Option<Map<String, String>> extraMetadata,
                              String commitActionType, Map<String, List<String>> partitionToReplaceFileIds,
                              Option<BiConsumer<HoodieTableMetaClient, HoodieCommitMetadata>> extraPreCommitFunc) {
+    return commitStats(instantTime, stats, Collections.emptyList(), extraMetadata, commitActionType,
+        partitionToReplaceFileIds, extraPreCommitFunc);
+  }
+
+  public boolean commitStats(String instantTime, List<HoodieWriteStat> dataTableStats, List<HoodieWriteStat> mdtStats,
```

Comment:
Let's also add javadocs here and make sure you're intentional about the scope of the methods so we can limit the points another developer can interact with to the smallest surface area. It will make it easier for us to maintain the code in the future

Reply:
sure. sg